codeclimate-wrapper

```diff
@@ -41,10 +41,18 @@ else
     && invalid_docker_host "$DOCKER_HOST"
 fi
 
-exec docker run \
-  --interactive --tty --rm \
-  --env CODE_PATH="$PWD" \
-  --volume "$PWD":/code \
-  --volume /tmp/cc:/tmp/cc \
-  --volume /var/run/docker.sock:/var/run/docker.sock \
-  codeclimate/codeclimate "$@"
+docker_run() {
+  exec docker run \
+    --rm \
+    --env CODE_PATH="$PWD" \
+    --volume "$PWD":/code \
+    --volume /tmp/cc:/tmp/cc \
+    --volume /var/run/docker.sock:/var/run/docker.sock \
+    "$@"
```

Comment:
I tested locally by running `./codeclimate-wrapper analyze > test.txt` and `./codeclimate-wrapper analyze`, and both worked correctly. That said, I'm pretty new to bash scripting - mostly started with Pat's suggestion above - so welcome other suggestions!